backend/src/main/kotlin/org/darkSolace/muse/statisticsModule/model/LastSeenEntry.kt

```diff
@@ -0,0 +1,23 @@
+package org.darkSolace.muse.statisticsModule.model
+
+import org.darkSolace.muse.userModule.model.User
+import java.util.*
+import javax.persistence.*
+
+/**
+ * The [LastSeenEntry] class containing visitor session information, a user if the visitor is
+ * logged in as well a timestamp when the visitor was last seen.
+ */
+@Entity
+class LastSeenEntry {
+    @Id
+    @GeneratedValue
+    val id: Long = 0
+
+    @OneToOne
+    var user: User? = null
+
+    @Temporal(TemporalType.TIMESTAMP)
+    var lastSeen: Date = Date()
+    var jSession: String? = null
+}
```

Comment:
The statistics module is new, right?
On the topic of modularity, maybe we could add an interface class and any non-core module can implement that statistics interface and implement their own statistics as needed?
Then the User module could have its own statistics thing and any other module as well?

Reply:
is changed in the refactor to "lastSeen" (#49)